GVRf/Sample/gvrf_controls_sample/assets/gvr_note4.xml

```diff
@@ -38,10 +38,10 @@
             colorFormat="COLOR_8888"
             depthFormat="DEPTH_24"
             fov-y="90.0"
-            multiSamples="1"
+            multiSamples="8"
```

Comment:
i should probably already know the answer to this, but do any of the current mobile chipsets support 8xMSAA?  what happens if you ask for 8xMSAA and it doesn't exist?